ruby/lib/ci/queue/redis/supervisor.rb

```diff
@@ -39,7 +39,19 @@ def wait_for_workers
             yield if block_given?
           end
 
-          puts "Aborting, timed out." if time_left <= 0 && !exhausted?
+          if time_left <= 0 && !exhausted?
+            puts "Aborting, timed out."
```

Comment:
Something like `Aborting, timed out waiting for the following tests to finish executing:` would make it a bit clearer.